docker/main/Dockerfile

```diff
@@ -174,6 +189,14 @@ RUN pip3 wheel --wheel-dir=/wheels -r /requirements-wheels.txt
 RUN --mount=type=bind,source=docker/main/install_hailort.sh,target=/deps/install_hailort.sh \
     /deps/install_hailort.sh
 
+# install pycoral & tflite runtime
```

Comment:
I don't think we should do this here, these should probably just be pulled in to the requirements-wheels file

Reply:
I can add an url to a wheel in requirements-wheels.txt?

Alternatively, should I add a script that's similar to `install_hailort.sh`, but for tflite_runtime?

Reply:
Yes, and we will want that URL to be conditioned so it installs based on the architecture. For example:

https://github.com/blakeblackshear/frigate/blob/6476f8ade78a337be2244ead486d68e6d6365807/docker/main/requirements-wheels.txt#L28-L29

we do not need a script like hailort because that installs the hailort firmware as well while we only are installing wheels